jdi-light-mobile/src/main/java/com/epam/jdi/light/mobile/elements/composite/AndroidScreen.java

```diff
@@ -45,4 +48,16 @@ public SystemBarInfo setRect(Rectangle rect) {
             return this;
         }
     }
+
+    static AndroidDriver driver= (AndroidDriver) getDriver();
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6336987)